go-controller/hybrid-overlay/pkg/controller/node_linux.go

```diff
@@ -399,23 +545,33 @@ func (n *NodeController) ensureHybridOverlayBridge() error {
 	}
 
 	portName := houtil.GetHybridOverlayPortName(n.nodeName)
-
-	// If the master hasn't yet created our hybrid overlay port don't
-	// return an error, but allow the caller to try again later
-	portMAC, portIP, _ := util.GetPortAddresses(portName)
-	if portMAC == nil || portIP == nil {
+	portMACString, haveDRMACAnnotation := node.Annotations[hotypes.HybridOverlayDRMAC]
+	if !haveDRMACAnnotation {
+		klog.Warningf("node %s does not have DRMAC annotation yet, failed to ensure hybrid overlay"+
+			"and will retry later", n.nodeName)
+		// node must not be annotated yet, retry later
```

Comment:
This shouldn't be a warning. The node watcher might sometimes see the other nodes before the master watcher does.